src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,38 @@
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, $card-width);
+  grid-template-rows: repeat(auto-fill, $card-height);
+  gap: 48px;
+  justify-content: center;
+
+  @media (max-width: 488px) {
+    grid-template-columns: $card-width;
+  }
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, $card-width);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, $card-width);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, $card-width);
+  }
+
+  &__card {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: $card-width;
+    height: $card-height;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+    margin-bottom: 48px;
```

Comment:
remove margins, you have gap 
<img width="497" alt="image" src="https://user-images.githubusercontent.com/66749919/208752672-c1144f6b-2f2b-46ff-9ddd-142e77db0870.png">